templates/server/config/environment/doNotCommit.env.js

```diff
@@ -0,0 +1,10 @@
+(function IIFE() {
+  'use strict';
+
+  // Environment variables and production values not to be comitted to repo but needed for development.
+  const config = {
+    // CLIENT_SECRET: 345912384fjwasif3p9w8fj (aws secret)
```

Comment:
So, these are not actually being added to `process.env`. Which means that all modules need to require the config file. It's fine, but some services (like [the auth service](https://github.com/Gesto/gesto-auth-middleware/blob/develop/lib/database/index.js#L10)) still look for env vars.